code/modules/clothing/masks/mystic.dm

```diff
@@ -1,75 +1,164 @@
+//todo:
+//ghost flickering + sprite
+//ghosts spooking near happiest mask causing an immediate flickering + happy boost
+
+#define VERYHAPPY 75
+
+#define HAPPIER 1
+#define HAPPIERBLOODY 4
+#define UNHAPPIER -10
+
 /obj/item/clothing/mask/happy
-	name = "Happiest Mask"
-	desc = "<span class=warning>\"I'm happy! I'M HAPPY! SEE! I SAID I'M HAPPY PLEASE DON'T\"<span>"
```

Comment:
Possibly because the mask only ever spawns renamed with the xenoarch find names (alien object, etc) and so they never read it?